File: app/src/main/java/org/deiverbum/app/data/entity/LHOfficePatristicJoinEntity.java
package org.deiverbum.app.data.entity;

import androidx.annotation.NonNull;
import androidx.room.ColumnInfo;
import androidx.room.Entity;
import androidx.room.PrimaryKey;

/**
 * @author A. Cedano
 * @version 1.0
 * @since 2022.2
 */

@Entity(tableName = "lh_office_patristic_join"
        //indices = {@Index(value = {"grupoFK","salmoFK"}, unique = true)}
        //primaryKeys = {"grupoFK","salmoFK"},
)
public class LHOfficePatristicJoinEntity {

    @NonNull
    @PrimaryKey
    @ColumnInfo(name = "groupID")
    public Integer grupoId=0;
}

File: app/src/main/java/org/deiverbum/app/data/entity/LHOfficePatristicJoinEntity.kt
package org.deiverbum.app.data.entity

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey
import org.deiverbum.app.util.Constants

/**
 * @author A. Cedano
 * @version 1.0
 * @since 2023.1
 */
@Entity(tableName = Constants.LH_OFFICE_PATRISTIC_JOIN)
class LHOfficePatristicJoinEntity {
    @PrimaryKey
    @ColumnInfo(name = "groupID")
    var grupoId = 0
}